model/post.go

```diff
@@ -31,6 +31,7 @@ type Post struct {
 	Id            string          `json:"id"`
 	CreateAt      int64           `json:"create_at"`
 	UpdateAt      int64           `json:"update_at"`
+	EditAt        int64           `json:"edit_at"`
```

Comment:
@jwilander, Actually the `OriginalId` always remains empty for a post. It is used to store `OriginalId` of a old post (before it's marked as deleted) and after a new post is created with same id, which happens when a post is updated. `OriginalId` is not stored in new post, it's only stored in old post. Refer code here: https://github.com/mattermost/platform/blob/ff127bbaa343026a55429acf37f2c95186d07ac5/store/sql_post_store.go#L127.